scripts/MKW_SaveGhostToBothCSV.py

```diff
@@ -0,0 +1,24 @@
+from dolphin import gui
+import TTK_Lib
+
+"""
+MKW_SaveGhostToBothCSV
+
+This script takes the current ghost's inputs and write them to the ghost and player csvs
+"""
+
+def main() -> None:
+    gui.add_osd_message("Script started")
+    
+    # Convert internal RKG to input list
+    input_sequence = TTK_Lib.readFullDecodedRKGData(TTK_Lib.PlayerType.GHOST)
+    
+    if (input_sequence is None or len(input_sequence) == 0):
+        gui.add_osd_message("Not in race")
+        return
+    
+    TTK_Lib.writeToCSV(input_sequence, TTK_Lib.PlayerType.GHOST)
+    TTK_Lib.writeToCSV(input_sequence, TTK_Lib.PlayerType.PLAYER)
```

Comment:
We probably want some sort of text confirming that the script ran successfully here (applies to all relevant files).
```suggestion
    TTK_Lib.writeToCSV(input_sequence, TTK_Lib.PlayerType.GHOST)
    TTK_Lib.writeToCSV(input_sequence, TTK_Lib.PlayerType.PLAYER)
    gui.add_osd_message("The ghost .rkg has been saved to both .csv files!")
```

Reply:
I added some error messages to let the user know if writes were successful or failed due to file lock.